spec/services/census_authorization_handler_spec.rb

```diff
@@ -63,10 +62,10 @@
       end
     end
 
-    xcontext "when it's someone who pays taxes in city" do
+    context "when it's someone who pays taxes in city" do
       let(:response_json) { CensusRestClient::StubbedResponseBuilder.build_not_resident_but_pays_taxes }
 
-      it { is_expected.to be_valid }
+      it { is_expected.not_to be_valid }
```

Comment:
so people that pay taxes are not authorized, correct?

Reply:
ouch...

Reply:
I found that: https://github.com/AjuntamentGava/decidim-gava-app/commit/07a9cbfecd006b15a94851976aa69626c39343f9